sdk/search/azure-search/pom.xml

```diff
@@ -43,12 +43,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
+      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
```

Comment:
Good catch here! Libraries built in different pipelines need to use dependency or unreleased dependencies if they need the current, built, version of the library.